tools.py

```diff
@@ -30,21 +31,22 @@ def subset(input_path, output_path, audio_name, align_boundary_words, cer, wer,
 		json.dump(transcript_cat, open(output_path, 'w'), ensure_ascii = False, sort_keys = True, indent = 2)
 	print(output_path)
 
-def cut(input_path, output_path, sample_rate, mono, dilate, strip, audio_backend):
+def cut(input_path, output_path, sample_rate, mono, dilate, strip, strip_prefix, audio_backend):
 	os.makedirs(output_path, exist_ok = True)
 	transcript_cat = []
 	
 	transcript = json.load(open(input_path))
 	prev_audio_path, signal = None, None
 
-	for t in sorted(transcript, key = lambda t: t['audio_path']):
+	for t in tqdm(sorted(transcript, key = lambda t: t['audio_path'])):
```

Comment:
`tqdm` is currently not used anywhere else. If you wish to track progress, please use `enumerate` and just `print(i, '/', len(transcript))` at the end of iteration